### ISSUE TITLE
About python trtc.Functor

### ISSUE THREAD

**Issue by zsy0828** [fynv/ThrustRTC#26]
hello,
I'm a new in learning thrust, and i'm really interesting about yout project, but I have some questions about  trtc.Functor.

I wonder can class `trtc.Functor` support curand_init() or rand() to generate a random number. 

i'd really appreciate if you reply me.
thank you very much!

**Comment by slayoo** [comment]
Please check out ThrustRTC's sister project CURandRTC: https://github.com/fynv/CURandRTC/

**Comment by zsy0828** [comment]
> Please check out ThrustRTC's sister project CURandRTC: https://github.com/fynv/CURandRTC/
Thank you very much !

**Comment by fynv** [comment]
Yes, CURandRTC provides a device-side implementation of XORWOW algorithm. 
That basically copies some code from curand_kernel.h
